Documentation/bpf/progtypes.rst

```diff
@@ -323,6 +323,13 @@ talked about XDP this mode is typically implied.
   production environment either the native or offloaded mode is better
   suited and the recommended way to run XDP.
 
+..
+
+* **Best Effort**
```

Comment:
This is generic, non-cilium specific, documentation about BPF and thus this doesn't belong here.

Reply:
Will remove.